src/unistd.rs

```diff
@@ -210,17 +215,22 @@ impl ForkResult {
 /// I'm a new child process
 /// ```
 ///
+///
 /// # Safety
+/// Make sure to read the [man page].
+/// Fork has some properties that can lead to unintended behavior.
+/// Especially the handling of mutexes and open files can be tricky.
 ///
 /// In a multithreaded program, only [async-signal-safe] functions like `pause`
-/// and `_exit` may be called by the child (the parent isn't restricted). Note
-/// that memory allocation may **not** be async-signal-safe and thus must be
+/// and `_exit` are safe to call by the child until it calls `execve`(the parent isn't restricted).
+/// Note that memory allocation may **not** be async-signal-safe and thus must be
 /// prevented.
 ///
 /// Those functions are only a small subset of your operating system's API, so
 /// special care must be taken to only invoke code you can control and audit.
 ///
 /// [async-signal-safe]: http://man7.org/linux/man-pages/man7/signal-safety.7.html
+/// [man page]: http://man7.org/linux/man-pages/man2/fork.2.html
```

Comment:
For standardized functions, it's better to use the open group's man page: http://pubs.opengroup.org/onlinepubs/9699919799/functions/fork.html